openstack_plugin/compat.py

```diff
@@ -473,6 +474,23 @@ def is_update_operation(self):
         return self.operation_name in [CLOUDIFY_UPDATE_OPERATION,
                                        CLOUDIFY_UPDATE_PROJECT_OPERATION]
 
+    def lookup_handler_for_openstack_node_v2(self):
+        """
+        This method will lookup the handler for openstack type v2 so that we
+        it is possible to continue transformation process to openstack
+        plugin v3
+        :return: Handler object in order to transform data to support
+        openstack version 3
+        """
+
+        for node_type in self._type_hierarchy:
+            handler_node = self.transformation_handler_map.get(node_type)
```

Comment:
Question about this method? Is there a chance that we could get the wrong node type?
Like let's say if someone uses `cloudify.openstack.nodes.WindowsServer` which is also derived from `cloudify.openstack.nodes.Server`?